src/domain/products/edit/sections/general/index.tsx

```diff
@@ -99,22 +99,39 @@ type DetailProps = {
 }
 
 const Detail = ({ title, value }: DetailProps) => {
+  const DetailValue = () => {
```

Comment:
suggestion:

```ts

if (Array.isArray(value) && value.length) {
  return <DelimitedList list={value} delimit={2} />
} else {
  return <p><p/>
}

return <p>{value ? value : "–"}</p>
```

Reply:
remember the `-` on line 4 😄

Reply:
Oho my bad i did made a mistake let me change it 😂

Reply:
Done @riqwan let me
Know what you think, it is nit so i let you choose

Reply:
It avoids sub if statements, makes more readable and easy to evolve